tests/providers/apache/hive/hooks/test_hive.py

```diff
@@ -383,6 +383,10 @@ def test_table_exists(self):
             self.hook.table_exists(str(random.randint(1, 10000)))
         )
 
+    def test_drop_partition(self):
+        self.assertTrue(self.hook.drop_partitions(self.table, db=self.database,
+                                                  part_vals=[DEFAULT_DATE_DS]))
+
```

Comment:
Does this tests requires some external service? Does it create side effects?

Reply:
@turbaszek Yes. it uses Hivemetastore Thrift client. it does the partition from the test table set up for unit testing.

Reply:
Do you think we can mock the client? In this way we will reduce the side effects and the test will not require any external service

Reply:
@turbaszek  Methods like def test_max_partition(self) also does the same. Moreover, it should not be an expensive operation. Do you think we really have to mock the call?

Reply:
Side effects in tests are not a good practise and Airflow already has number of flaky tests. So I would be in favor of using mocking.

Reply:
Sounds fair. Let me get this changed :)